.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug
+about: Something does not work as expected
+----
+
+### Summary
+
+<!-- provide a summary here -->
+
+### Affected System
+
```

Comment:
Maybe we should explain them atleast what is expected here like in the other blocks as well